### PR TITLE
Release version 0.10.0

### PR DESCRIPTION
**Pull request by jasagredo**
Builds on top of #49

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+#### 0.10.0 (2024-05-27)
```
Comment (stevana) [inline CHANGELOG.md:1]: 2024-05-28

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+* Revert `WithSetup` variants that were introduced in 0.7.2. Due to the way
+  repetitions are done now, there is no need for these variants anymore (PR #43);
+
+* Delete `run[N]ParallelCommandsNTimes` in favor of the new
+  `forAll[N]ParallelCommandsNTimes` as repetitions now happen at the `forAll`
+  level (PR #43);
+
+* Rename `forall` to `forAll` (PR #49).
+
+* Replace `ansi-wl-pprint` with `prettyprinter` (PR #49).
```
Comment (stevana) [inline CHANGELOG.md:12]: Same here, explain why: i.e. because ansi-wl-pprint is deprecated in favour of prettyprinter.

File: CHANGELOG.md (hunk 1 above)
Comment (jasagredo) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+#### 0.10.0 (2024-05-27)
+
+* Revert `WithSetup` variants that were introduced in 0.7.2. Due to the way
+  repetitions are done now, there is no need for these variants anymore (PR #43);
+
+* Delete `run[N]ParallelCommandsNTimes` in favor of the new
+  `forAll[N]ParallelCommandsNTimes` as repetitions now happen at the `forAll`
+  level (PR #43);
+
+* Rename `forall` to `forAll` (PR #49).
```
[the remark this answers is not in the document]
Comment (jasagredo) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (jasagredo) [reply CHANGELOG.md]: Done

**Review by stevana — APPROVED** [review]

**Pull request merged**